compiler/one-cmds/onecc

```diff
@@ -52,7 +52,7 @@ def _call_driver(driver_name, options):
     with subprocess.Popen(
             cmd, stdout=subprocess.PIPE, stderr=subprocess.STDOUT, bufsize=1) as p:
         for line in p.stdout:
-            sys.stdout.buffer.write(line)
+            sys.stdout.buffer.write(f"{driver_name}: ".encode() + line)
```

Comment:
I have two questions.

Although irrelevant to this PR, is there any special reason for connecting stderr to stdout on line 52?

Regarding this PR, isn't it correct that the error message is displayed in the module where the error message occurred?

Reply:
> Although irrelevant to this PR, is there any special reason for connecting stderr to stdout on line 52?

That is a recommend way to implement `subprocess` in [document](https://docs.python.org/3/library/subprocess.html).

> If capture_output is true, stdout and stderr will be captured. When used, the internal Popen object is automatically created with stdout=PIPE and stderr=PIPE. The stdout and stderr arguments may not be supplied at the same time as capture_output. If you wish to capture and combine both streams into one, use stdout=PIPE and stderr=STDOUT instead of capture_output.

> isn't it correct that the error message is displayed in the module where the error message occurred?

I think it is doing as you said. Current `one-cmds` tools spit out their outputs. That outputs should be piped(`stdout=subprocess.PIPE`) to be displayed.

Reply:
@lemmaa Thank you for review. For 1st question, @jyoungyun got answer no special reason..., and she will follow up this issue. For 2nd question, I think it depends on. Anyway, I will continue to create PR or issue to help users identify where the error comes.

Reply:
> ... If you wish to capture and combine both streams into one ...

@mhs4670go , Is there any reason for it? If not, it is natural to bypass stderr of called process to stderr of caller process. 

> I think it is doing as you said. Current `one-cmds` tools spit out their outputs. That outputs should be piped(`stdout=subprocess.PIPE`) to be displayed.

My point in here is `onecc` prepends `driver-name` to the error message of called process. :)

Reply:
@lemmaa  
> Is there any reason for it?

Both way looks okay to me. The reason why I chose the way I did it is because I wanted to keep the log sequences.

Say, the error logs are like this sequence.

```python3
print("First", file=sys.stderr)
print("Second", file=sys.stdout)
print("third", file=sys.stderr)
```

Then, current implementation prints like this.
```
First
Second
Third
```

But, if I split the pipe(`stdout=subprocess.PIPE`, `stderr=subprocess.PIPE`), I'm going to have to catch them separately. And, it will print like this.
```
Second # print stdout first
First
Thrid
```